src/sync/branches.ts

```diff
@@ -6,25 +6,28 @@ import { BackfillMessagePayload } from "~/src/sqs/sqs.types";
 
 // TODO: better typings
 export const getBranchTask = async (
-	logger: Logger,
+	parentLogger: Logger,
 	gitHubClient: GitHubInstallationClient,
 	_jiraHost: string,
 	repository: Repository,
 	cursor: string | undefined,
 	perPage: number,
 	messagePayload: BackfillMessagePayload) => {
 
-	logger.info("Syncing branches: started");
+	const logger = parentLogger.child({ backfillTask: "Branch" });
```

Comment:
The app already has in the parent sync/installation.ts. I feel this line is like duplication. 
```
		const logger = parentLogger.child({
				task: nextTask
			});

			logger.info("Starting task");
			nextTask.startTime = Date.now();
```

Reply:
It does, but theres a few steps between that log and the actual execution, the parent log here is to tag all the other logs in the function a bit better, the `task:nextTask` attribute we inherit is an object and its a pain to Splunk with.

Otherwise its pretty much a tidier log :shrug:

Reply:
Inside the `const taskExecutor = async (nextTask: Task` in the parent ( as in my above comments ), the `logger.info("Starting task");` actually means the start of the task, I don't see any extra steps?

Reply:
I'll mark this as /nit for now to unblock you. But let me know if you can think of anyway to improve.